src/tools/run-make-support/src/lib.rs

```diff
@@ -97,6 +101,35 @@ impl AuxBuildInvocationBuilder {
     }
 }
 
+#[derive(Debug)]
+pub struct RustdocInvocationBuilder {
```

Comment:
nit: this can just be called `Rustdoc`, I have a pending PR that simplifies `RustcInvocationBuilder` to `Rustc` (no need for such a long name).